NGitLab.Mock.Tests/PipelineTests.cs

```diff
@@ -15,11 +15,33 @@ public async Task Test_pipelines()
             var commit = project.Repository.Commit(user, "test");
 
             var pipeline = project.Pipelines.Add(commit.Sha, JobStatus.Success, user);
+
             var job = pipeline.AddNewJob("Test_job", JobStatus.Success);
             job.Trace = "This is a trace\nWith Multiple line";
 
+            var reportSummary = pipeline.TestReportsSummary = new Models.TestReportSummary
```

Comment:
In this case, can it have its own test? That would discourage adding more in this Test_pipelines test

Reply:
seperate test created :)